listenbrainz/webserver/static/js/src/user/Listens.tsx

```diff
@@ -687,19 +688,30 @@ export default class Listens extends React.Component<
               />
             )}
             <ListenCountCard user={user} listenCount={listenCount} />
-            {user && (
-              <div
-                className="card hidden-xs hidden-sm"
-                style={{ paddingTop: "1.5em" }}
-              >
-                <UserSocialNetwork user={user} newAlert={newAlert} />
-              </div>
-            )}
+            {user && <UserSocialNetwork user={user} newAlert={newAlert} />}
           </div>
           <div className="col-md-8 col-md-pull-4">
             {!listens.length && (
-              <div className="lead text-center">
-                <p>No listens yet</p>
+              <div className="empty-listens">
+                <FontAwesomeIcon icon={faRecordVinyl as IconProp} size="10x" />
+                <div className="lead empty-text">Empty record</div>
```

Comment:
Somehow this doesn't quite sound right.I like the play on words though, so I'm a little torn.
I keep playing it in my head (like a record) and I'm not convinced, but I don't have any good suggestion.
Maybe "Get started" ?

Reply:
In line with that, I didn't fully recognize a vinyl icon.
I have a few suggestions for icons I think would work well in context:
- https://fontawesome.com/icons/music-slash?s=solid
- https://fontawesome.com/icons/user-music?s=solid 
- https://fontawesome.com/icons/album-collection-circle-plus?s=solid 
- maybe weird? https://fontawesome.com/icons/head-side-headphones?s=solid

Reply:
Dang,  I didn't realized those were all "pro" icons…
Welp, perhaps just the CD then? https://fontawesome.com/icons/compact-disc?s=solid

I'm with you on "get started" being a bit meh. 
Following your suggestion of "get playing", after some though I think "Start listening" or "Get listening" would work well (considering we call the pieces of data "listens")

Reply:
I like the CD as well! :)

And "Start listening" goes really well with Listens. I'll do the changes according to this then :)